src/initialization/InitDistribution.H

```diff
@@ -17,6 +17,8 @@
 
 #include <utility>  // for std::move
 
+// Declare the external variable
+extern amrex::ParticleReal bunch_charge;
```

Comment:
We don't use global variables - let me know if you still need this and we'll pass it around or get it from ParmParse or the particle container :)
```suggestion
```